_tech-blog/2020-08-31-gsoc-20-resume.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: GSoC 20 - Automatized support for Organised Editing Guidelines
+date: 2020-08-31 12:30:00 Z
+Summary Text: Here is the resume from the GSoC' 20 project that adds
+  automatized support for Organised Editing Guidelines in
+  Tasking Manager.
+Feature Image: "/uploads/gsoc-20-architecture.jpg"
+Person:
+- Wille Marcel
+---
+
+
+# GSoC 20 - Automatized support for Organised Editing Guidelines
```

Comment:
you can remove that line, the title is already on the top of the page

Reply:
Removed this